CHANGELOG.txt

```diff
@@ -12,6 +12,7 @@ Current Development
 - Adding TerminationCondition and SolverStatus to pyomo.environ (#429)
 - Resolved problems with ExternalFunctions with fixed arguments (#442)
 - Adding the Pyomo5 expression system, which supports PyPy (#272)
+- Reworking unhashable indexing (#467)
```

Comment:
This is editorial, but we didn't rework indexing.  Could you make it "Efficiency improvements when processing unhashable component indices"?